### PR TITLE
Contact form pair programming

### PR DESCRIPTION
**Pull request by berylcl**
In this project I was able to :
&check;Add a contact form.
&check;Use the [Formspree service](https://formspree.io/html).

### CODE REVIEW

File: portfolio.html [hunk 1]
```diff
@@ -206,18 +207,63 @@ <h2>About Me</h2>
...
     </ul>
-
 </section>
-
-
+<section class="form">
+    <p>
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+    </p>
+    <form action="https://formspree.io/f/{form_id}" method="post">
```
Comment (ghost) [inline portfolio.html]: - [ ] Currently, your form submission raises an error. Kindly create a form with [formspree service](https://formspree.io/html) and use the unique id generated in your `action` prop.
 
image

File: portfolio.html [hunk 2]
```diff
@@ -206,18 +207,63 @@ <h2>About Me</h2>
...
+        <!-- <p>
+            <label for="username">
+            <input id="username" type="text" name="username" placeholder="Full Name" maxlength="30" required>
+        </p>
+        <p><label for="email"></label>
+            <input id="email"  name="email" type="email"placeholder="Email Address" required>
+        </p> -->
```
Comment (ghost) [inline portfolio.html]: - [ ] Kindly remove the commented-out code to keep your code base clean.

File: portfolio.html [hunk 3]
```diff
@@ -206,18 +207,63 @@ <h2>About Me</h2>
...
+            <li>
+                <label for="username">
+                <input id="username" type="text" name="username" placeholder="Full Name" maxlength="30" required>
+            </li>
+            <li>
+                <label for="email"></label>
+                <input id="email"  name="email" type="email"placeholder="Email Address" required>
+            </li>
+            <li>
+                <label for="textarea"></label><textarea name="textarea" id="textarea" rows="7" cols="100" maxlength="500" placeholder="Write me something..."></textarea>
```
Comment (ghost) [inline portfolio.html]: - [ ] Kindly add an attribute `required` to  your `<textarea>` element for form validation

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

✅ Correct use of Github workflow
✅ No linter error


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: portfolio.html [hunk 4]
```diff
@@ -206,18 +207,52 @@ <h2>About Me</h2>
...
+            <li>
+                <label for="username">
+                <input id="username" type="text" name="username" placeholder="Full Name" maxlength="30" required>
+            </li>
+            <li>
+                <label for="email"></label>
+                <input id="email"  name="email" type="email"placeholder="Email Address" required>
+            </li>
+            <li>
+                <label for="textarea"></label><textarea name="message" id="textarea" rows="7" cols="100" maxlength="500" placeholder="Write me something..."></textarea>
```
Comment (leonardodiasb) [inline portfolio.html]: - [ ] As the previous code reviewer mentioned and the project required, kindly add the `require` attribute to the `textarea` too. The user cannot submit the form if any of the input fields are empty.
 - 
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Gakiiberyl, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

✅ Correct use of Github workflow
✅ No linter error

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Olamarx — APPROVED** [review]
Hi @Gakiiberyl, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linters passed ✔️ 
- Commit messages ✔️

## Optional suggestions
- [OPTIONAL] I would implore you to add some features of your app and not the tech stack to your Key Features and your Future Features. Also, remove the demo under the tech stack.
image


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**